3_javascript/javascript1/week1/A dog age calculator.js

```diff
@@ -0,0 +1,11 @@
+const dogYearOfBirth= 2017;
+const dogYearFuture= 2027;
+const humanAge= dogYearFuture - dogYearOfBirth;
+const dogAge= 7*humanAge;
+let shouldShowResultInDogYears;
+if (shouldShowResultInDogYears= true){
+ console.log( "Your dog will be " + dogAge + " dog years old in " + dogYearFuture );
+}
+else{
+    console.log( "Your dog will be " + humanAge + " human years old in " + dogYearFuture );
+}
```

Comment:
🐶 Your logic to calculate dog age and human age is spot on!
🚩 Be cautious with your if condition. You're using a single equals sign (=) which is an assignment, not a comparison. You'll want to use `===` to check for equality:
`if (shouldShowResultInDogYears === true)`
🔧 Currently, `shouldShowResultInDogYears` is not initialized to any value. It would be best if you initialize it to either true or false based on the desired behavior.